spec/features/transfer_account_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'transfer user account ownership', type: :feature do
+  let(:user) { create(:user, :with_jwt) }
+  let(:form_dom_id) { "#transfer_account_user_#{user.id}" }
+  let(:email) { 'valid@email.com' }
+
+  subject do
+    visit user_profile_path
+    within form_dom_id do
+      fill_in 'email', with: email
+      click_button 'Confirmer'
```

Comment:
```suggestion
      click_button
```

ça devrait suffire vu que c'est le seul bouton

Reply:
Yep ça passe, parfait.